malwarefront/src/components/ShowObjectPresenter.js

```diff
@@ -101,9 +96,10 @@ export default class ShowObjectPresenter extends Component {
         let ObjectAction = (props) => {
             return (
                 <li className="nav-item">
-                    <Link to="#" 
+                    <Link to={props.link ? props.link : "#"}
                           className="nav-link" 
-                          onClick={(ev) => {ev.preventDefault(); props.action()}}>
+                          onClick={(ev) => {props.action && props.action()}}
```

Comment:
We can shorten it a bit
```suggestion
                          onClick={() => props.action && props.action()}
```